client/src/modules/stock/exit/exit.js

```diff
@@ -676,8 +734,11 @@ function StockExitController(
 
     return Stock.movements.create(movement)
       .then(document => {
-        ReceiptModal.stockExitLossReceipt(document.uuid, bhConstants.flux.TO_LOSS);
-        reinit(form);
+
+        if (document.uuid) {
```

Comment:
This could happen if two users connect to the Stock Exit interface and the first user consumes the entire stock of a product before the second validates its consumption, and as the control is also carried out at server level, an error message will be returned to the client without there being a document.uuid